test/test_t2star.py

```diff
@@ -202,24 +202,23 @@ def test_t2star_run_end2end(self):
                     dt_factor=dt_factor,
                 )
 
-            expdata = exp.run(
-                backend=backend,
-                shots=2000,
-            )
-            result = expdata.analysis_result(0)
-            self.assertAlmostEqual(
-                result["t2star_value"],
-                estimated_t2star * dt_factor,
-                delta=3 * dt_factor,
-            )
-            self.assertAlmostEqual(
-                result["frequency_value"],
-                estimated_freq / dt_factor,
-                delta=3 / dt_factor,
-            )
-            self.assertEqual(
-                result["quality"], "computer_good", "Result quality bad for unit " + str(unit)
-            )
+                # run circuits
+                exp.set_analysis_options(shots=2000)
```

Comment:
`shots` is supposed to be a run option and not an analysis option. So I guess that you just run with the default of 1024 shots.

Reply:
You are right. Fixed.